package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "buzz-word-bingo",
+  "version": "0.0.1",
+  "description": "tracking buzz words",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/zbuzzeo/Buzz-Word-Bingo.git"
+  },
+  "author": "github.com/zbuzzeo",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/zbuzzeo/Buzz-Word-Bingo/issues"
+  },
+  "homepage": "https://github.com/zbuzzeo/Buzz-Word-Bingo#readme",
+  "dependencies": {
+    "body-parser": "^1.18.3",
+    "express": "^4.16.4",
+    "nodemon": "^1.18.9",
+    "npm": "^6.7.0"
```

Comment:
Not sure why you added npm as a dep, it doesn't need to be here. Same for nodemon.